src/Services/DatabaseBackup/MysqlDatabaseBackup.php

```diff
@@ -84,7 +78,7 @@ public function backup(AbstractExecutor $executor): void
         $executor->getReferenceRepository()->save($this->getBackupFilePath());
         self::$metadata = $em->getMetadataFactory()->getLoadedMetadata();
 
-        exec("mysqldump -h $dbHost -u $dbUser -p$dbPass --no-create-info --skip-triggers --no-create-db --no-tablespaces --compact $dbName > {$this->getBackupFilePath()}");
+        exec("MYSQL_PWD=$dbPass mysqldump --host $dbHost --port=$dbPort --user $dbUser --no-create-info --skip-triggers --no-create-db --no-tablespaces --compact $dbName > {$this->getBackupFilePath()}");
```

Comment:
Using an environment variable remove this error: `Warning: Using a password on the command line interface can be insecure.`.